VEHICLE/SetRandomTrains.md

```diff
@@ -8,7 +8,31 @@ ns: VEHICLE
 void SET_RANDOM_TRAINS(BOOL toggle);
 ```
 
+Enables spawning random trains on the preset tracks. However to be able to use this call you would first need to call at least two other calls to make it function properly.
```

Comment:
After re-reading this it could be reworded to include the natives' require for it to function in the sentence.